ferrilata-devilline/Controllers/PitchController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using ferrilata_devilline.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace ferrilata_devilline.Controllers
+{
+
+    [Route("api")]
+    public class PitchController : Controller
+    {
+        public PitchController()
+        {
+        }
+
+        [HttpPost("post/pitch")]
+        public IActionResult PostPitch([FromBody] Pitch NewPitch)
+        {
+            var request = Request;
+            var headers = request.Headers;
+
+            if (!headers.ContainsKey("Authorization") ||
+                headers["Authorization"].ToString().Length == 0)
+            {
+                return Unauthorized(new { message = "Unauthorized" });
+            }
+
+
+            if (NewPitch == null ||
```

Comment:
This expression could be checked by a separate method

Reply:
done;